catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/geofeature/GazetteerFeatureService.java

```diff
@@ -62,15 +60,11 @@ public void setFeatureQueryable(FeatureQueryable featureQueryable) {
   @Override
   public List<String> getSuggestedFeatureNames(String query, int maxResults) {
     try {
-      return geoEntryQueryable
-          .query(query, maxResults)
-          .stream()
-          .map(GeoEntry::getName)
-          .collect(toList());
+      return geoEntryQueryable.getSuggestedNames(query, maxResults);
     } catch (GeoEntryQueryException e) {
-      LOGGER.debug("Error while making feature service request.", e);
-      return Collections.emptyList();
+      LOGGER.debug("Suggestion query failed", e);
```

Comment:
`LOGGER.debug("Suggestion query failed. {}", e);`

Reply:
This log the stack trace of the throwable without the `{}`

Reply:
So used to always needing to add curly braces in. That is valid slf4j API usage